### ISSUE TITLE
Support CallAs on AWS::CloudFormation::StackSet

### ISSUE THREAD

**Issue by cptcanuck** [aws-cloudformation/cfn-lint#2060]
*cfn-lint version: (`cfn-lint --version`)* 

```
> docker run -v $PWD:/cfn -it cfn-python-lint:latest "--version"
cfn-lint 0.40.0
```

*Description of issue.*

currently cfnlint says 

```
E3002 Invalid Property Resources/AWSConfigStackSet/Properties/CallAs
```

But CallAs is a valid property:

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cloudformation-stackset.html#cfn-cloudformation-stackset-callas

Please provide as much information as possible:
* Template linting issues:
  * Please provide a CloudFormation sample that generated the issue.

```yaml
AWSTemplateFormatVersion: '2010-09-09'
Description: This CloudFormation StackSet the AWS provided stack that enables AWS Config with a centralized S3 bucket for storage
Resources:
  AWSConfigStackSet:
    Type: AWS::CloudFormation::StackSet
    Properties:
      CallAs: DELEGATED_ADMIN
      TemplateURL: https://cloudformation-stackset-sample-templates-us-east-1.s3.us-east-1.amazonaws.com/EnableAWSConfigForOrganizations.yml
      Parameters:
        - ParameterKey: Frequency
          ParameterValue: '3hours'
        - ParameterKey: S3BucketName
          ParameterValue: !Ref ConfigBucketName
        - ParameterKey: SNS
          ParameterValue: False
      PermissionModel: SERVICE_MANAGED
      AutoDeployment:
        Enabled: True
        RetainStacksOnAccountRemoval: False
      StackInstancesGroup:
        - DeploymentTargets:
            OrganizationalUnitIds:
              - !Ref TargetOU
          Regions: !Ref deployRegions
      OperationPreferences:
          FailureToleranceCount: 0
          MaxConcurrentCount: 5
      StackSetName: StackSet-Enable-AWS-Config-In-Organization
      Description: Stackset that deploys Config to configured regions in the organization
```

Cfn-lint uses the [CloudFormation Resource Specifications](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) as the base to do validation. These files are included as part of the application version. Please update to the latest version of `cfn-lint` or update the spec files manually (`cfn-lint -u`)

**Comment by PatMyron** [comment]
[`0.40.0`](https://github.com/aws-cloudformation/cfn-lint/releases/tag/v0.40.0) was released in 2020

Upgrade to more recent versions for support for the latest CloudFormation types
